package.json

```diff
@@ -17,6 +17,7 @@
     "postinstall": "yarn tsc",
     "prettier": "prettier --config .prettierrc --write \"**/*.{js,ts}\"",
     "prettier:diff": "prettier --config .prettierrc --list-different \"**/*.{js,ts}\"",
+    "prettier-docs": "prettier --config .prettierrc --write \"**/*.md\"",
```

Comment:
Not in precommit / ci ?

How about mdx

Reply:
> Not in precommit/ci?

I was undecided about this. But I guess adding to precommit is ok as long as it doesn't fail the CI because of misformatting.

> How about mdx?

I initially tried it but there's an issue https://github.com/prettier/prettier/issues/7041 where Prettier doesn't work with MDX. It's fixed in https://github.com/prettier/prettier/pull/6949 but not released yet. When we upgrade Prettier to the next release we can try including support for prettifying MDX.